.github/workflows/ci.yml

```diff
@@ -284,6 +284,57 @@ jobs:
           cache-from: type=registry,ref=ghcr.io/fuellabs/fuel-core-build-cache:latest
           cache-to: type=registry,ref=ghcr.io/fuellabs/fuel-core-build-cache:latest,mode=max
 
+  # duplicate of publish-docker-image, but with profiling features enabled
+  # this is split into a separate action since it takes longer to build
+  publish-docker-image-profiling:
+    needs:
+      - verifications-complete
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ghcr.io/fuellabs/fuel-core-profiling
+          tags: |
+            type=sha
+            type=ref,event=branch
+            type=ref,event=tag
+            type=semver,pattern={{raw}}
+          flavor: |
+            latest=${{ github.ref == 'refs/heads/master' }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to the ghcr.io registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push the image to ghcr.io
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: deployment/Dockerfile
+          build-args: |
+            "FEATURES=production,profiling"
+            "DEBUG_SYMBOLS=true"
```

Comment:
I didn't know that `ARG FEATURES="production"` and `ARG DEBUG_SYMBOLS=false` will not override env variables if you set them before=) Nice to know